shared/styles/status-bar.ios.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import {StatusBarIOS, NativeModules} from 'react-native'
+const {StatusBarManager} = NativeModules
+
+const addSizeListener = (cb: Function) => {
+  // gcall with initial value
```

Comment:
gcall -> call (or did you mean something else?)

Reply:
nope that's a typo